docs/7.x/changelog.md

```diff
@@ -1274,6 +1274,26 @@ to learn how to gain insight into how the cluster status changes over time.
 
 ## 5.5 Releases
 
+### 5.5.52 LTS (September 8th, 2020)
+
+#### Improvements
+
+* Make sure that agents are available before resuming or rolling back an upgrade ([#2044](https://github.com/gravitational/gravity/pull/2044)).
+* Restart flannel before upgrade from older versions to workaround potential overlay issues ([#2043](https://github.com/gravitational/gravity/pull/2043)).
+* Remove the requirement for all nodes to have the same operating system version from preflight checks ([#2048](https://github.com/gravitational/gravity/pull/2048)).
+* Display active Kapacitor alerts in `gravity status` ([#2039](https://github.com/gravitational/gravity/pull/2039)).
+* Update `gravity status` to display a warning upon state directory ownership change ([#2034](https://github.com/gravitational/gravity/pull/2034), [planet#727](https://github.com/gravitational/planet/pull/727), [satellite#255](https://github.com/gravitational/satellite/pull/255)).
+* Validate `HTTP_PROXY` and `HTTPS_PROXY` variables set via `RuntimeEnvironment` resource ([#2049](https://github.com/gravitational/gravity/pull/2049)).
+* Update the log collector rotation policy to keep ten 256MB log files ([#2033](https://github.com/gravitational/gravity/pull/2033), [logging#70](https://github.com/gravitational/logging-app/pull/70)).
+* Drop excessive metrics from InfluxDB during an upgrade ([#2091](https://github.com/gravitational/gravity/pull/2091), [monitoring#187](https://github.com/gravitational/monitoring-app/pull/187)).
+
+#### Bugfixes
+
+* Fix an issue with remote_syslog process leaking CPU time ([#2084](https://github.com/gravitational/gravity/pull/2084), [logging#71](https://github.com/gravitational/logging-app/pull/71)).
+* Fix an issue with etcd not being upgraded on the worker nodes ([#2086](https://github.com/gravitational/gravity/issues/2068), [planet#738](https://github.com/gravitational/planet/pull/738)).
```

Comment:
I think this describes the wrong solution - the change removed the upgrade step on worker nodes so that they would simply start with the latest available etcd?

Reply:
Yeah well what you described is an implementation detail. The outcome for users is that etcd on workers wasn't being upgraded and now that's fixed.